### PR TITLE
Amendments for usage guidelines.

### PR DESCRIPTION
**Pull request by yanxiaodi**

### CODE REVIEW

**Comment by LisandroSu** [comment]
Hello, @yanxiaodi 

Thank for your contribution. We are currently reviewing it. We will communicate the decision taken  accordingly. 
 
Thanks for your time and support on this.

Kind regards,

Microsoft C&E Localization Team

File: docs/standard/design-guidelines/equality-operators.md [hunk 1]
```diff
@@ -19,31 +19,31 @@ ms.lasthandoff: 10/06/2018
 ms.locfileid: "48839350"
 ---
 # <a name="equality-operators"></a>相等运算符
-本部分讨论了重载相等运算符，并将 `operator==` 和 `operator!=` 作为相等运算符。
+本部分讨论了重载相等运算符并将 `operator==` 和 `operator!=` 引用为相等运算符。 
 
-**X 切忌** 重载相等运算符的其中一个而不重载另一个。  
+**X 请勿**重载这两个相等运算符的任意一个。
```
Comment (yanxiaodi) [inline docs/standard/design-guidelines/equality-operators.md]: "X DO NOT overload one of the equality operators and not the other."
前面提到相等运算符包括`operator==` 和 `operator!=` ，我感觉此处的意义为，不要只重载其中的一个，要么就将这两个都重载，要么就都不重载，如果翻译为“请勿重载这两个相等运算符的任意一个”，意义有点混淆，容易理解成“这两个相等运算符是不能重载的”。但实际上是可以重载的，只是如果要重载的话，必须同时重载这两个。谢谢。

File: docs/standard/design-guidelines/equality-operators.md [hunk 2]
```diff
@@ -19,31 +19,31 @@ ms.lasthandoff: 10/06/2018
...
+本部分讨论了重载相等运算符并将 `operator==` 和 `operator!=` 引用为相等运算符。 
 
-**X 切忌** 重载相等运算符的其中一个而不重载另一个。  
+**X 请勿**重载这两个相等运算符的任意一个。 
 
-**✓ 务必** 确保 <xref:System.Object.Equals%2A?displayProperty=nameWithType> 和相等运算符具有完全相同的语义和类似的性能特性。
+**✓ 请确**保 <xref:System.Object.Equals%2A?displayProperty=nameWithType> 和相等运算符具有完全相同的语义和类似的性能特性。
 
-这通常意味着重载相等运算符时 `Object.Equals` 也需要被重写。  
+这通常意味着，在重载相等运算符时需要替代 `Object.Equals`。
```
Comment (yanxiaodi) [inline docs/standard/design-guidelines/equality-operators.md]: 一般`overload`翻译成`重载`，`override`翻译成`重写`。

File: docs/standard/design-guidelines/attributes.md [hunk 3]
```diff
@@ -15,36 +15,36 @@ ms.contentlocale: zh-CN
...
 换句话说，不要使用可同时用构造函数和 setter 设置的属性。此原则使哪些参数是可选的、哪些参数是必需的变得非常明确，并可避免使用两种方法执行相同的操作。  
 
-**X 避免** 重载自定义属性的构造函数。
+**X 避免**重载自定义特性构造函数。
 
-只有一个构造函数可以清楚地向用户传达哪些参数是必需的，哪些是可选的。
+只使用一个构造函数来清晰地向用户传达哪些参数是必需的，哪些是可选的。
 
-**✓ 务必** 尽可能密封自定义属性类。这使属性的查找速度更快。
+**✓ 务必**尽可能密封自定义特性类。这可使查找属性的速度更快。
```
Comment (yanxiaodi) [inline docs/standard/design-guidelines/attributes.md]: 这可使查找属性的速度更快。=> 这可使查找**特性**的速度更快。

File: docs/standard/design-guidelines/guidelines-for-collections.md [hunk 4]
```diff
@@ -13,117 +13,117 @@ ms.lasthandoff: 09/17/2018
...
+如果你确定唯一需提供支持的情况是仅正向迭代，则只需使用 `IEnumerable<T>` 即可。
 
-**✓ 考虑** 使用泛型基本集合的子类而不是直接使用集合。  
+**✓ 请考**虑使用通用基本集合的子类，而不是直接使用集合。 
 
-这允许使用更好的名称以及添加基本集合类型中不存在的辅助成员。这尤其适用于高级 API。
+通过此操作，可使用更好的名称并添加基本集合类型中不存在的辅助成员。这尤其适用于高级 API。
 
-**✓ 考虑** 从非常常用的方法和属性返回 `Collection<T>` 或 `ReadOnlyCollection<T>` 的子类。  
+**✓ 请考**虑从非常常用的方法和属性返回 `Collection<T>` 或 `ReadOnlyCollection<T>` 的子类。
```
Comment (yanxiaodi) [inline docs/standard/design-guidelines/guidelines-for-collections.md]: Could we keep '考虑' as one word, not separate it. So I think the better one is **考虑** or **请考虑**, but not **请考**虑. The other similar situations include **更倾**向, **请使**用, etc. Thanks.

**Comment by LisandroSu** [comment]
Hello, @yanxiaodi 

Thank you for your feedback and suggestions.
 
This proposed file change has been accepted. The suggestion will be merged to this repository.  We will release this article on the next localization release round. 
 
Many thanks for your support.

Kind regards,

Microsoft C&E Localization Team

**Pull request merged**